airflow-core/src/airflow/ui/src/layouts/Details/PanelButtons.tsx

```diff
@@ -98,7 +115,33 @@ export const PanelButtons = ({ dagView, setDagView, ...rest }: Props) => {
         </IconButton>
       </ButtonGroup>
       <Stack alignItems="flex-end" gap={1} mr={2}>
-        <DagVersionSelect disabled={dagView !== "graph"} />
+        <HStack>
+          <DagVersionSelect disabled={dagView !== "graph"} />
+          {dagView === "grid" ? (
+            <Select.Root
+              bg="bg"
+              collection={displayRunOptions}
+              data-testid="display-dag-run-options"
+              onValueChange={handleLimitChange}
+              size="sm"
+              width="200px"
+            >
+              <Select.Trigger>
+                <Select.ValueText placeholder="Display Dag Run Limit" />
```

Comment:
```suggestion
                <Select.ValueText placeholder="Total Dag Runs" />
```

Reply:
It would probably be better for us to have 10 selected by default instead of a placeholder or the clear button